youtube_dl/extractor/nova.py

```diff
@@ -152,14 +171,30 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, display_id)
 
+        description = clean_html(self._og_search_description(webpage, default=None))
+        if site == 'novaplus':
+            upload_date = unified_strdate(self._search_regex(
+                r'(\d{1,2}-\d{1,2}-\d{4})$', display_id, 'upload date', default=None))
+        elif site == 'fanda':
+            upload_date = unified_strdate(self._search_regex(
+                r'<span class="date_time">(\d{1,2}\.\d{1,2}\.\d{4})', webpage, 'upload date', default=None))
+        else:
+            upload_date = None
+
         # novaplus
         embed_id = self._search_regex(
-            r'<iframe[^>]+\bsrc=["\'](?:https?:)?//media\.cms\.nova\.cz/embed/([^/?#&]+)',
+            r'<iframe[^>]+\bsrc=[\"\'](?:https?:)?//media\.cms\.nova\.cz/embed/([^/?#&]+)',
             webpage, 'embed url', default=None)
+
         if embed_id:
-            return self.url_result(
+            info = {
+                'description': description,
+                'upload_date': upload_date
+            }
+            info.update(self.url_result(
                 'https://media.cms.nova.cz/embed/%s' % embed_id,
-                ie=NovaEmbedIE.ie_key(), video_id=embed_id)
+                ie=NovaEmbedIE.ie_key(), video_id=embed_id))
+            return info
```

Comment:
My main issue is that I don't know how to make the 'url result' type to inherit the description and upload type from  the "upper" container. The test will fail if I uncomment 'description' and 'upload_date' in the TEST dictionary...
But when I run the youtube-dl  with my modifications, all seem to work ok.

Reply:
You should use info dict with `'_type': 'url_transparent'` instead.

Reply:
addressed -- now passes the test with flying colors.